packages/main/src/plugin/extension/extension-loader.ts

```diff
@@ -1773,7 +1773,7 @@ export class ExtensionLoader {
       // delete the path
       if (extension.removable) {
         await fs.promises.rm(extension.path, { recursive: true, force: true });
-      } else {
+      } else if (!extension.devMode) {
         throw new Error(`Extension ${extensionId} is not removable`);
```

Comment:
remark: we lose the fallback here ? shouldn't we have instead `if (extension.removable && !extension.devMode)` at L1774 ?

What are the condition where `removable` is false ? because maybe removable should be false when we are in `devMode`